op/model.py

```diff
@@ -72,6 +71,38 @@ def __init__(self, name, backend, cache):
 
         self._is_our_app = self.name == self._backend.app_name
 
+        self._status = None
+
+    @property
+    def status(self):
+        if not self._is_our_app:
+            return UnknownStatus()
+
+        if not self._backend.is_leader():
+            raise RuntimeError('cannot get application status as a non-leader unit')
+
+        if self._status:
+            return self._status
+
+        s = self._backend.status_get(True)
```

Comment:
Anti-pattern: a reader cannot tell what the True means without going over to the status_get definition and reading the code. We need to force this to be spelled as foobar=True, so that we can read it.

Reply:
Agreed.